docs/source/sharing.rst

```diff
@@ -0,0 +1,106 @@
+Sharing
+=======
+
+Plone comes with a sophisticated user management system that allows to assign users and groups with global roles and permissions. Sometimes this in not enough though and you might want to give users the permission to access or edit a specific part of your website or a specific content object. This is where local roles (located in the Plone sharing tab) come in handy.
+
+
+Retrieve Local Roles
+--------------------
+
+In plone.restapi, the representation of any content object will include a hypermedia link to the local role / sharing information in the 'sharing' attribute:
+
+.. code:: json
```

Comment:
@sneridagh @tisto that's ok, I'll do that in a second pass where I convert all remaining literal code examples to ones generated by `test_documentation.py` and displayed using `sphinxcontrib-httpexample`.